lib/wovnrb.rb

```diff
@@ -104,7 +103,21 @@ def switch_lang(body, values, url, lang=@store.settings['default_lang'], headers
       body.close if body.respond_to?(:close)
       new_body
     end
-  end
 
+    private
+
+    # Checks if a given HTML body is an Accelerated Mobile Page (AMP).
+    # To do so, it looks at the required attributes for the HTML tag:
+    # https://www.ampproject.org/docs/tutorials/create/basic_markup.
+    #
+    # @param {Nokogiri::HTML5::Document} body The HTML body to check.
+    #
+    # @returns {Boolean} True is the HTML body is an AMP, false otherwise.
+    def is_amp_page?(body)
+      html_attributes = body.xpath('//html')[0].try(:attributes) || {}
+
+      !html_attributes['amp'].nil? || !html_attributes["\u26A1"].nil?
```

Comment:
You can also write `html_attributes['amp'] || html_attributes["\u26A1"]`, that is more simple.

Reply:
I changed, thanks